tsconfig.json

```diff
@@ -18,7 +18,7 @@
         "jsx": "react",
         "lib": [
             "dom",
-            "es2020"
+            "es2021"
```

Comment:
As discussed in chat, and https://github.com/cockpit-project/cockpit/pull/21391/commits/f3e61ec1ff164a6c239111f1f176016fb2efeb26 : let's not mix code with different standards, too confusing. Let's move everything to 2021 consistently. I'm also ok with 2022 to finally match our .eslintrc.